src/components/ServiceInstallation/Tabs/Services.vue

```diff
@@ -97,32 +114,25 @@
           "totaljs-flow",
           "totaljs-messageservice",
         ],
+        servicesToCheckTypeDB: [
```

Comment:
can you please update the variable name to something meaningful ?

servicesToCheckBeforeMovingToDashboard or something like that

Reply:
Done